build/root/Makefile

```diff
@@ -624,4 +624,5 @@ else
 bazel-release:
 	bazel build //build/release-tars
 	bazel shutdown
+	pkill ^bazel || true
```

Comment:
should we kill all the java processes that are bazel leftovers?

https://github.com/bazelbuild/bazel/issues/1868
https://github.com/bazelbuild/bazel/issues/4505#issuecomment-406312017

Reply:
> bazel shutdown stops workers only from the current workspace.

we only have one workspace. though it seems shutdown isn't cutting it ...?